README.md

```diff
@@ -229,13 +319,23 @@ UIKit lets you align labels by top, bottom, or baseline. BonMot includes `BONTex
 
 You can use `BONTextAlignmentConstraint` programmatically or in Interface Builder. In code, use the convenience initializer:
 
+```swift
+BONTextAlignmentConstraint(item: someLabel,
+                      attribute: BONConstraintAttribute.CapHeight,
```

Comment:
#pp I think I'd prefer just `.CapHeight` here. What do you think, @nevillco? Is it clear what's going on without it? Does it need to be, in this case?